BenchmarkTest00229.py

```
@@ -0,0 +1,15 @@
+#{fact rule=untrusted-deserialization@v1.0 defects=0}
+
+from django.http import HttpResponse
+import datetime
+
+def current_datetime(request):
+    # ok:avoid-insecure-deserialization
+    user_obj = request.cookies.get('uuid')
```

Ruff (0.13.1)

Local variable `user_obj` is assigned to but never used

Remove assignment to unused variable `user_obj`

(F841)